developers/development/bindings.md

```diff
@@ -5,43 +5,30 @@ title: Developing bindings
-* You can choose `org.openhab`, if you want it to be used for openHAB only.
-This is the better option, if your binding is not interesting for other solutions, requires special libraries or has technical dependencies on openHAB specific things (although this should be avoided as much as possible).
-
-## Creating a Skeleton
-
-For the openHAB namespace: Choose the option "openHAB 2 Add-ons" in [your IDE setup](ide.html), and create a skeleton for your binding.
-To do this, go into your Git repository under `git/openhab2-addons/addons/binding` and call the script `create_openhab_binding_skeleton.sh` with two arguments.
+Choose the option "openHAB 2 Add-ons" in [your IDE setup](ide.html), and create a skeleton for your binding.
+To do this, use a command line terminal and go to your Git repository folder under `git/openhab2-addons/addons/binding` and call the script `create_openhab_binding_skeleton.sh` with two arguments.
 It is important that your binding name is in camel case (e.g. 'ACMEProduct' or 'SomeSystem').
 After the binding name, provide your name as the author (surrounded by quotes if you want to use whitespaces to separate your first and last name).
 Example: `~/git/openhab2-addons/addons/binding/create_openhab_binding_skeleton.sh BindingName "Firstname Lastname"`
 
-For the Eclipse SmartHome namespace: Choose the option "Eclipse SmartHome Extensions" in [your IDE setup](ide.html), and create a skeleton for your binding.
-To do this, go to `git/smarthome/tools/archetype`and run `mvn install` in order to install the archetype definition in your local Maven repo.
-Now go to `git/smarthome/extensions/binding` and call the script `create_binding_skeleton.sh` with two parameters.
-The first one is your binding name in camel case (e.g. 'ACMEProduct' or 'SomeSystem').
-The second one is your name as author (surrounded by quotes if you want to use whitespaces to separate your fist and last name).
-
-Now switch back to Eclipse and choose `File->Import->General->Existing Projects into Workspace`, enter the folder of the newly created skeleton as the root directory, and press "Finish".
+Now go back to the IDE and choose `File->Import->General->Existing Projects into Workspace`, enter the folder of the newly created skeleton as the root directory, and press "Finish".
 
 _Note:_ Here you can find a [screencast of the binding skeleton creation](http://youtu.be/30nhm0yIcvA).
 
+## Before You Start
+
+If you plan to contribute your binding to openHAB through a PR, here is what we suggest to help keeping the potential effort for changes due to reviews low:
+
+- Before you start with the implementation, create an issue on Github and talk about your plan to come up with a PR. Maybe there are others that have privately a similar binding before and would be ready to contribute it. Also, it might help finding collaborators for the implementation.
```

Comment:
```suggestion
- Before you start with the implementation, create an issue on GitHub and talk about your plan to come up with a PR. Maybe there are others that have privately a similar binding before and would be ready to contribute it. Also, it might help finding collaborators for the implementation.
```